packages/theme/src/cli/commands/theme/push.ts

```diff
@@ -149,22 +151,24 @@ export default class Push extends ThemeCommand {
       return
     }
 
-    const developmentThemeManager = new DevelopmentThemeManager(adminSession)
```

Comment:
`DevelopmentThemeManager` marks all themes as development - any theme created via the push command will be marked as [yours].

To address this, I've created a `ThemeManager` and renamed the base class to `BaseThemeManager`